src/lib/form/abstractFormNode.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { FormManager } from "./formManager";
+import { ValidationRule } from "./validators";
+
+type ControlValue = string | Record<string, AbstractFormNode>;
+
+/**
+ * This is the base class for `FieldManager`, and `FieldManager`.  It represents a node in the form tree of depth 1.
```

Comment:
```suggestion
 * This is the base class for `FieldManager`, and `FormManager`.  It represents a node in the form tree of depth 1.
```
Teeny tiny thing

Reply:
@Ninamma Thank you for spotting. I have updated!